R/rdeephaven/R/client_wrapper.R

```diff
@@ -1,8 +1,18 @@
+#' @description
+#' A Client is the entry point for interacting with the Deephaven server. It is used to create new tables,
+#' import data to and from the server, and run scripts on the server.
 #' @export
 Client <- R6Class("Client",
   cloneable = FALSE,
   public = list(
     .internal_rcpp_object = NULL,
+
+    #' @description
+    #' Initializes a Client object that connects to the Deephaven server.
+    #' This method can accept either a string or an Rcpp::XPtr object, and will call
+    #' `initialize_for_target` or `initialize_for_xptr` accordingly.
+    #' @param ... String denoting the address of a running Deephaven server, formatted as `"ip:port"`,
+    #' or an Rcpp::XPtr object that points to an existing client connection.
```

Comment:
This is confusing and unclear.  The "string" path can take a ton more args than just a string.  This needs to get reworked to be very clear and precise.

Reply:
You're right, but those args are documented in the "initialize_from_target" doc. I would rather not have this method exposed at all and wish users could think of it as an overloaded constructor. I don't really know how to document this properly.